docker-compose.nod.yml

```diff
@@ -1,12 +1,16 @@
 version: '2.1'
+
+networks:
+  nod-newscrawler:
+
 services:
   crawler:
-    build: .
+    image: uhhlt/newscrawler
```

Comment:
@seyyaw I think the image should still be build from source. Otherwise each change requires that the images is first pushed to Docker Hub

Reply:
We can build images with the Dockerfile. One can build the image from the source first using the Dockerfile. The docker-compose file is really handy to run images rather then build them.

May be if it is important, you can add how to build the image using DockerFile in the documentation. `docker build -t IMAGENAME .`

Reply:
Yes,  running `docker build -t uhhlt/newscrawler .` is a solution. Previously I just used `docker-compose up --build` to be sure to always use the latest version. It's just a minor point for me. I've also now added a .dockerignore file to improve the build speed and prevent permission errors when accessing the data in nod-data